src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const populationSpans = document.querySelectorAll('.population');
+
+const totalPopulation = [...populationSpans]
+  .reduce((sum, population) => {
+    return sum + Number(population.innerText.replace(/,/g, ''));
+  }, 0);
+
+const averagePopulation = totalPopulation / populationSpans.length;
+
+const totalSpan = document.querySelector('.total-population');
+const averageSpan = document.querySelector('.average-population');
+
+totalSpan.innerText = totalPopulation.toLocaleString('en-US');
+averageSpan.innerText = averagePopulation.toLocaleString('en-US');
```

Comment:
IMHO, `textContent` is preferable. [That's why.](https://stackoverflow.com/a/35213639)